truenas_install/__main__.py

```diff
@@ -434,6 +466,21 @@ def main():
 
                 write_progress(0.5, "Performing post-install tasks")
 
+                for entry in TRUENAS_DATASETS:
+                    if not (force_mode := entry.get("mode")):
+                        continue
+
+                    os.chmod(f"{root}/{entry['name']}", force_mode)
+
+                # /usr will be readonly and so we want the ca-certificates directory to symlink
+                # to writeable location in /var/local
+                os.makedirs(f"{root}/usr/local/share", exist_ok=True)
+                shutil.rmtree(f"{root}/usr/local/share/ca-certificates", ignore_errors=True)
```

Comment:
Shouldn't this be done in rootfs image preparation stage instead?

Reply:
Yeah, that's a better place to put it.